### DISCUSSION TITLE
[Kavita] Allow turning page using PageUp/PageDown and Up/Down arrow keys

### DISCUSSION THREAD

**Discussion by jason3232** [Kareadita/Kavita#3421] [Ideas]
### Idea Description

Currently, Kavita's web reader only accept turning pages using Left/Right arrow keys while using the 1 Column or 2 Columns Layout Modes. 

It would be nice to also support PageUp/PageDown keys and the Up/Down arrow keys for turning pages.

For context, I was trying to read books on Kavita with a Boox Page using the EinkBro browser and the page turn buttons are not working as they were sending PageUp/PageDown and Up/Down keys. I think the same issue would applies to other devices/setup using the volume keys to turn as well.

### Idea Category

User Experience

### Duration of Using Kavita

1 month

### Before submitting

- [X] I've already searched for existing ideas before posting.

**Comment by majora2007** [comment]
I missed this FR, but this is done. Kavita rolled out the ability to bind any key to any action.
